src/lib/y2storage/proposal/space_maker.rb

```diff
@@ -413,13 +470,14 @@ def delete_disk_content(planned_partitions, lvm_helper, disk)
         log.info "BEGIN delete_disk_content with disk #{disk}"
 
         disks_for(new_graph, disk).each do |dsk|
-          break if success?(planned_partitions)
           log.info "Checking if the disk #{dsk.name} has a partition table"
 
-          if dsk.has_children? && dsk.partition_table.nil?
-            log.info "Found something that is not a partition table"
-            remove_content(dsk, lvm_helper)
-          end
+          next unless dsk.has_children? && dsk.partition_table.nil?
```

Comment:
that's only for rubocop's pleasure

Reply:
well, it makes some sense, to show only what is interesting, so early skip non-interesting parts. Maybe I just will write it like
```
next unless dsk.has_children?
next if dsk.partition_table
# so we have children but no partition table
```

Reply:
Well, **I** would have kept it the way it was. rubocop only started complaining due to a change someplace else.

Reply:
it complain, because success was moved. So now only problematic disk is processed and rest can be early quit ( theory say that early quits in code safe some brain processing, but in reality it helps only sometimes and sometimes make it worse ).